x/shield/abci.go

```diff
@@ -4,10 +4,16 @@ import (
 	abci "github.com/tendermint/tendermint/abci/types"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/certikfoundation/shentu/common"
+	"github.com/certikfoundation/shentu/x/shield/types"
 )
 
 // BeginBlock executes logics to begin a block.
 func BeginBlock(ctx sdk.Context, req abci.RequestBeginBlock, k Keeper) {
+	if ctx.BlockHeight() == common.UpdateHeight {
```

Comment:
Let's not call it "UpdateHeight". How about "Update1Height"?

Reply:
OK SGTM